src/hb-ot-shape-complex-khmer.cc

```diff
@@ -389,11 +389,13 @@ reorder_khmer (const hb_ot_shape_plan_t *plan,
 	       hb_font_t *font,
 	       hb_buffer_t *buffer)
 {
-  insert_dotted_circles_khmer (plan, font, buffer);
-
-  foreach_syllable (buffer, start, end)
-    reorder_syllable_khmer (plan, font->face, buffer, start, end);
+  if (buffer->message (font, "start reordering khmer")) {
```

Comment:
I’d put the name of the shaper inside brackets, `start reordering (khmer)`, but I have no strong opinion about it.

Reply:
Or do away with the shaper name, it is not included in the other messages.

Reply:
I don't think I'll do that - in #2261 we had a discussion about consistency of message formats to ensure easy parsing and Behdad said

> We're not fully consistent, but aiming for "action item-type identifier" where action is start here, item-type is table, and identifier is GSUB.

So that's what I'm aiming for here too.

Reply:
OK, it isn’t making things more inconsistent than they already are,